DcShare/Helper/ShareAttachment.swift

```diff
@@ -63,6 +66,37 @@ class ShareAttachment {
         }
     }
 
+    // for now we only support GIF
+    func createAnimatedImageMsg(_ item: NSItemProvider) {
```

Comment:
`private`  ... (I guess all `create...`-methods in this file can be private).

Reply:
fixed